js/actions/windowActions.js

```diff
@@ -213,6 +213,9 @@ const WindowActions = {
     if (frameOpts.location && UrlUtil.isURL(frameOpts.location)) {
       frameOpts.location = UrlUtil.getUrlFromInput(frameOpts.location)
     }
+    // default the title to the location
+    frameOpts.title = frameOpts.title || frameOpts.location
```

Comment:
When a new tab/window is created with an opener the location isn't sent initially. We create a new about:blank tab and when it's ready the opener sets the location. I am now sending the "delayedLoadUrl" with the new window/tab event so we can display the url until the page is loaded and the real title is available.